supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesRuntimeAttributesSpec.scala

```diff
@@ -274,6 +277,28 @@ class JesRuntimeAttributesSpec extends WordSpecLike with Matchers {
       val workflowOptions = workflowOptionsWithDefaultRA(Map(MemoryKey -> JsString("65 GB")))
       assertJesRuntimeAttributesSuccessfulCreation(runtimeAttributes, workflowOptions, expectedRuntimeAttributes)
     }
+
+    "use reasonable default values" in {
+      val expectedRuntimeAttributes = staticDefaultsWithUbuntu
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { docker: "ubuntu:latest" }""").head
+      assertJesRuntimeAttributesSuccessfulCreation(runtimeAttributes, emptyWorkflowOptions, expectedRuntimeAttributes)
+    }
+
+    "warn for unrecognized keys" in {
+      val runtimeAttributes = createRuntimeAttributes(HelloWorld, """runtime { docker: "ubuntu:latest" whatIsThis: "noIdea" andThis: "donno" }""").head
+      val mockLogger = mock[Logger]
+      mockLogger.warn(anyString).answers { _ match {
+        case message: String =>
+          // The order cannot be guaranteed because runtime attributes come as an unordered map
+          // So manually check for keys independently
+          message should include("Unrecognized runtime attribute keys:")
+          message should include("whatIsThis")
+          message should include("andThis")
+      }
+      }
```

Comment:
I'm going to lash out since @cjllanwarne poked fun at my spacing earlier today. This can't be the correct indentation :)

Don't worry Chris, you're still going to bear the full brunt of my riled up-ness